packages/zql/src/query/query.ts

```diff
@@ -172,6 +172,7 @@ export interface Query<
     name: string,
     args: ReadonlyArray<ReadonlyJSONValue>,
   ): Query<TSchema, TTable, TReturn>;
+  delegate(delegate: QueryDelegate): Query<TSchema, TTable, TReturn>;
```

Comment:
The delegate is tied to the Zero instance so I don't think it is needed on the Query interface.